### ISSUE TITLE
[BUG] System.UnauthorizedAccessException: Access to the path 'Arimo-Bold.ttf' is denied.

### ISSUE THREAD

**Issue by Daxt1boss** [bloxstraplabs/bloxstrap#3408]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [X] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

2,8.0

### What problem did you encounter?

![image](https://github.com/user-attachments/assets/de5b774e-d76a-4496-8b9f-4ce5c2ba0b6e)

i idk, played roblox, at the moment its lagged, its was really 0 fps. Ima opend again and it died.
(don't ask why it russian)
also idkk its last version or no

### Bloxstrap Log

```text
2024-10-22T22:07:27Z [App::OnStartup] Starting Bloxstrap v2.8.0
2024-10-22T22:07:27Z [App::OnStartup] Compiled Wednesday, 16 October 2024 at 12:44:37 PM from commit 42ffec26cf2e080fe22210305a08fcb286aeb577 (tag/v2.8.0)
2024-10-22T22:07:27Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-10-22T22:07:27Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-10-22T22:07:27Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-10-22T22:07:27Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241022T220727Z.log
2024-10-22T22:07:27Z [Logger::Initialize] Finished initializing!
2024-10-22T22:07:27Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-10-22T22:07:27Z [JsonManager<Settings>::Load] Loaded successfully!
2024-10-22T22:07:27Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-10-22T22:07:27Z [JsonManager<State>::Load] Loaded successfully!
2024-10-22T22:07:27Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-10-22T22:07:27Z [FastFlagManager::Load] Loaded successfully!
2024-10-22T22:07:29Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-10-22T22:07:29Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-10-22T22:07:29Z [Bootstrapper::Run] Running bootstrapper
2024-10-22T22:07:29Z [Bootstrapper::SetStatus] Подключение к Roblox...
2024-10-22T22:07:29Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-10-22T22:07:29Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-10-22T22:07:29Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-10-22T22:07:29Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-10-22T22:07:29Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-10-22T22:07:29Z [Bootstrapper::Run] Connectivity check finished
2024-10-22T22:07:29Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-10-22T22:07:29Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-22T22:07:30Z [Bootstrapper::CheckForUpdates] No updates found
2024-10-22T22:07:30Z [Bootstrapper::GetLatestVersionInfo] Got channel as zflagged-httpbuf
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://bloxstraplabs.com/metrics/post?key=robloxChannel&value=zflagged-httpbuf
2024-10-22T22:07:30Z [Deployment::GetInfo] Getting deploy info for channel zflagged-httpbuf
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/zflagged-httpbuf
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessResponse] 429 Too Many Requests https://bloxstraplabs.com/metrics/post?key=robloxChannel&value=zflagged-httpbuf
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer/channel/zflagged-httpbuf
2024-10-22T22:07:30Z [Deployment::GetInfo] Getting deploy info for channel production
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-22T22:07:30Z [RegistryKeyEx::SetValueSafe] Writing 'zflagged-httpbuf' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-10-22T22:07:30Z [ApplicationSettings::Fetch.PCClientBootstrapper.zflagged-httpbuf] Fetching fast flags
2024-10-22T22:07:30Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/settings/application/PCClientBootstrapper/bucket/zflagged-httpbuf
2024-10-22T22:07:31Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/settings/application/PCClientBootstrapper/bucket/zflagged-httpbuf
2024-10-22T22:07:31Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/channel/common/version-678af0760dd24f8a-rbxPkgManifest.txt
2024-10-22T22:07:31Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/channel/common/version-678af0760dd24f8a-rbxPkgManifest.txt
2024-10-22T22:07:31Z [Bootstrapper::SetStatus] Устанавливаем {product}...
2024-10-22T22:07:31Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-10-22T22:07:31Z [LaunchHandler::LaunchRoblox] An exception occurred when running the bootstrapper
2024-10-22T22:07:31Z [App::FinalizeExceptionHandling] System.UnauthorizedAccessException: Access to the path 'Arimo-Bold.ttf' is denied.
   at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)
   at System.IO.FileSystem.RemoveDirectory(String fullPath, Boolean recursive)
   at Bloxstrap.Bootstrapper.UpgradeRoblox()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
```

**Comment by Wanwood1** [comment]
![Screenshot_5439](https://github.com/user-attachments/assets/f27fd4fc-55be-460f-81f0-862ff131fad3)
Funny, reinstall maybe it will help

**Comment by pizzaboxer** [comment]
Fixed in 2.8.1 (assuming so, reopen if not)
